tests/smoke-go-group-rules.yaml

```diff
@@ -0,0 +1,437 @@
+                  requirements:
+                    - file: go.mod
+                      groups: []
+                      requirement: v1.4.0
+                      source:
+                        source: rsc.io/quote
+                        type: default
+                  version: 1.4.0
+            dependency_files:
+                - /go/go.mod
+                - /go/go.sum
+    - type: create_pull_request
+      expect:
+        data:
+            base-commit-sha: 624ba6716c1d387a68b5a4c24fa25212b8f28f1d
+            dependencies:
+                - name: github.com/fatih/color
+                  previous-requirements:
+                    - file: go.mod
+                      groups: []
+                      requirement: v1.7.0
+                      source:
+                        source: github.com/fatih/color
+                        type: default
+                  previous-version: 1.7.0
+                  requirements:
+                    - file: go.mod
+                      groups: []
+                      requirement: 1.15.0
+                      source:
+                        source: github.com/fatih/color
+                        type: default
+                  version: 1.15.0
+                - name: github.com/inconshreveable/mousetrap
+                  previous-requirements:
+                    - file: go.mod
+                      groups: []
+                      requirement: v0.0.0-20141017200713-76626ae9c91c
+                      source:
+                        source: github.com/inconshreveable/mousetrap
+                        type: default
+                  previous-version: 0.0.0-20141017200713-76626ae9c91c
+                  requirements:
+                    - file: go.mod
+                      groups: []
+                      requirement: 1.1.0
+                      source:
+                        source: github.com/inconshreveable/mousetrap
+                        type: default
+                  version: 1.1.0
+                - name: golang.org/x/crypto
+                  previous-requirements:
+                    - file: go.mod
+                      groups: []
+                      requirement: v0.0.0-20180617042118-027cca12c2d6
+                      source:
+                        source: golang.org/x/crypto
+                        type: default
+                  previous-version: 0.0.0-20180617042118-027cca12c2d6
+                  requirements:
+                    - file: go.mod
+                      groups: []
+                      requirement: 0.9.0
+                      source:
+                        source: golang.org/x/crypto
+                        type: default
+                  version: 0.9.0
+            updated-dependency-files:
+                - content: |
+                    module github.com/dependabot/vgotest
+
+                    go 1.12
+
+                    require (
+                    	github.com/fatih/color v1.7.0
```

Comment:
Looks like color wasn't actually bumped.

Reply:
Good catch. Strange because it thinks it gets bumped [here](https://github.com/dependabot/smoke-tests/pull/68/files#diff-fc1e85da9e8cb84e1d9ebbdcbebcb4a02a1871406520fadeca2e7ed4ae26f6f9R115) and [here](https://github.com/dependabot/smoke-tests/pull/68/files#diff-fc1e85da9e8cb84e1d9ebbdcbebcb4a02a1871406520fadeca2e7ed4ae26f6f9R236). Worth investigating.